man/smooth-methods.Rd

```diff
@@ -26,7 +27,8 @@
 
       Smooths all spectra in \code{MSnExp}. \code{method} could be
       \code{"SavitzkyGolay"} or
```

Comment:
Please remove the "or" and replace it by a comma, reads: SavitzkyGolay, MovingAverage or MovingWeightedAverage.